modules/record_collection_data/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "values" {
+  description = "Cloudflare Zone DNS Records"
+  value = {
+    "${cloudflare_record.this_data.data[0].flags} ${cloudflare_record.this_data.data[0].tag} ${cloudflare_record.this_data.data[0].value}" = {
```

Comment:
@danvaida could we probably use underscores(`_`) here as separators? Otherwise it looks kind of wierd in the output:

```
      + CAA_somedomain.example.com        = {
          + 0 issue someca.com = {
              + created = (known after apply)
              + id      = (known after apply)
            }
        }
```

Reply:
We can, of course.
And we should.
Every key should be something that can be looked up, but if you check the current implementation, it does the same thing with SPF records.
This way it is consistent.

Reply:
I see... that's true. This is a tricky one. In case of SPF/TXT records it has only value and if we replace `" "` with `_` we could have collisions. Let's say - you have two TXT records which have values:

- `word1 word2`
- `word1_word2`

if you replace spaces with undersocres you will have a collision. On the other hand - with `data` blocks you all the time in control of content and the key.

I don't see it as an huge issue. For consistency reasons - ignore my initial comment.